dataflow/src/main/java/org/checkerframework/dataflow/busyexpression/BusyExprValue.java

```diff
@@ -0,0 +1,49 @@
+package org.checkerframework.dataflow.busyexpression;
+
+import org.checkerframework.checker.nullness.qual.Nullable;
+import org.checkerframework.dataflow.analysis.AbstractValue;
+import org.checkerframework.dataflow.cfg.node.Node;
+import org.checkerframework.javacutil.BugInCF;
+
+/** A busy expression wrapper turning a node into the abstract value */
+public class BusyExprValue implements AbstractValue<BusyExprValue> {
+
+    /**
+     * A busy expression is represented by a node, which can be a {@link
+     * org.checkerframework.dataflow.cfg.node.BinaryOperationNode}
+     */
+    protected final Node busyExpression;
+
+    @Override
+    public BusyExprValue leastUpperBound(BusyExprValue other) {
+        throw new BugInCF("lub of BusyExpValue get called!");
```

Comment:
```suggestion
        throw new BugInCF("leastUpperBound should never be called in a backwards analysis");
```

Reply:
Is that the reason why this method should never be called, because this is a backwards analysis?

Reply:
I understand it in very busy expression analysis, we only need to compute the lub of two stores (that is, 2 sets of BusyExpValue), and don't need to compute the lub of two abstract values from different stores. I think it's not because it's backwards analysis

In ConstantPropagation demo, they implemented the lub method to calculate the results of the two values.

Reply:
Because of this, I think it's better not throw any error but do nothing in the function. But I am not sure

Reply:
Last time we discusses that because `BusyExprValue` doesn’t use the `leastUpperbound` method, these seems no need for ` BusyExprValue ` to implement the `AbstractValue` interface. I removed it but found that `RegularTransferResult` class needs a class to implement the `AbstractValue` interface.